qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/RunReportPage.java

```diff
@@ -31,7 +32,12 @@ public void enterStartDate(String START_DATE) {
 	}
 
 	public void enterEndDate(String END_DATE) {
-		waiter.until(ExpectedConditions.elementToBeClickable(END_DATE_FIELD));
```

Comment:
Why don't you want to wait until the element is clickable?

Reply:
I found it another method of getting away with expected conditions rather used the javascript executor.

Reply:
Reverted it back to use clickabe

Reply:
So now, as it is, how is this function entering the end date?

Reply:
Its entering end date right here https://github.com/openmrs/openmrs-contrib-qaframework/blob/c09bf627d5b08b7c809876f5f4671345083d17a1/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ReportSteps.java#L77 after entering start date.

Reply:
Then what is the use of this method?

Reply:
Basicaly Running a report requires two significant dates, the first date, and the end date.

Reply:
So what exactly does this method do?

Reply:
This method waits for the element of end date to be clickable, After its clickabe then continues to allow RunReportPage.enterDate() to run for the workflow with static end date.

Reply:
How is that related to the method name?

Reply:
Let me update it by adding setTextToFieldNoEnter(END_DATE_FIELD, END_DATE);